build.gradle

```diff
@@ -5,60 +5,68 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.0'
+        classpath 'com.android.tools.build:gradle:4.2.0'
     }
 }
 
+plugins {
+    id "de.undercouch.download" version "4.1.1"
+}
+
 apply plugin: 'com.android.application'
 
+// Load keystore properties
+def keystorePropertiesFile = rootProject.file("keystore.properties")
+def keystoreProperties = new Properties()
+keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
```

Comment:
Hi Raju, I guess you took this pattern from my code.  I'm wondering that we need to update our public docs as well since it is documented with the local.properties file usage.

Reply:
Yeah, I like your approach of keeping it in separate properties files. I think it will be good to change the public docs. Just need your suggestion. I am okay with keeping it in a single gradle.properties file also.